Patcher/UnityInstallation.cs

```diff
@@ -0,0 +1,80 @@
+namespace Patcher
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text.RegularExpressions;
+
+    public class UnityInstallation
+    {
+        private readonly string _installationLocation;
+
+        public UnityInstallation(string installationLocation)
+        {
+            this._installationLocation = installationLocation;
```

Comment:
`this` isn't strictly needed here. I prefer to avoid it unless explicitly required.